examples/bpk-component-map/examples.js

```diff
@@ -395,6 +455,8 @@ export {
   WithPriceMarkersExample,
   WithPriceMarkersV2Example,
   WithIconPriceMarkersV2Example,
+  WithPriceMarkersV2ButtonExample,
+  WithIconPriceMarkersV2ButtonExample,
```

Comment:
Using PriceMarkersV2Button directly without a map is not what we expect, and it is not the purpose of exporting this component, so I don’t think we should add these two examples, but we should add example of using this component to interact with Popover on Maps.

Reply:
Yes, i have change to using this component to interact with Popover on Maps.